projects/packages/forms/src/blocks/step/editor.scss

```diff
@@ -0,0 +1,37 @@
+// Frontend styles for the form-step block
+.jetpack-form-step__break {
+	text-transform: uppercase;
+	margin: 10px 0;
+	text-align: center;
+
+	&::after {
+		content: '';
+		display: block;
+		width: 100%;
+		height: 1px;
+		background-color: #ddd;
+		margin-top: -12px;
+	}
+
+	.jetpack-form-step__label {
+		background-color: #FFF;
```

Comment:
Needs to use variable/theme styles

Reply:
no this is more of a UI component.

Reply:
Currently the page break component doesn't use any base-stypes ot theme colours.

Reply:
Gotcha 👍  In that case you could follow [paywall block's styles](https://github.com/Automattic/jetpack/blob/54cdd85da5dc9d9b9aae1f2fef1d815b5d0d7f31/projects/plugins/jetpack/extensions/blocks/paywall/editor.scss#L1-L42): we specifically designed it to be improvement over core's page break, which wasn't a great example to follow. It was more important to look good (and similar enough) than to be consistent with core in that case.

Note how we used variables there for things like font size.